Dockerfile

```diff
@@ -12,6 +12,9 @@ WORKDIR /usr/src/app
 COPY Gemfile Gemfile.lock ./
 RUN bundle install --without dev
 
+RUN pip install flask
+RUN pip install logic
```

Comment:
@wilsonwong1990 can you help me understand why this is needed?

Reply:
Yeah! When I setting up my own code spaces we can't use test.py without logic installed.

Reply:
🤔 Were you getting an error of some kind? When I was looking at the tests here https://github.com/es-na-battlesnake/snakes/pull/173/files#diff-e784e09539c3775677aa46db6cdae7f99baf3487476267d7a9bb7ce351e99e39R11 I was under the impression that this was importing the `logic.py` file so we could reference the methods it contains. That logic would be different from what I think `pip install logic` would install.

Reply:
Working with @bevns looks like there was some error in my one codespace. We can remove `logic`